pulp_ansible/app/tasks/collections.py

```diff
@@ -597,7 +597,15 @@ async def _post_save(self, batch):
                     fileobj=artifact_file, mode="r"
                 ) as tar:
                     log.info(_("Reading MANIFEST.json from {path}").format(path=artifact.file.name))
-                    file_obj = tar.extractfile("MANIFEST.json")
+                    for manifest in ["MANIFEST.json", "./MANIFEST.json"]:
```

Comment:
What case causes us to also check `./MANIFEST.json` ?

Reply:
If you look at the bug, there's one collection version tar that has `./MANIFEST.json`:

```
❯ tar -ztvf rshad-collection_demo-1.0.2.tar.gz                               
-rw-r--r-- rashz/rashz   10373 2019-11-15 11:14 ./CHANGELOG.md
-rw-r--r-- rashz/rashz   34129 2019-11-15 12:11 ./FILES.json
-rw-r--r-- rashz/rashz   24736 2019-11-15 11:14 ./LICENSE
-rw-r--r-- rashz/rashz     876 2019-11-15 12:11 ./MANIFEST.json
...
```

Reply:
idk how tar can have `./MANIFEST.json`, but I thought it would not be cool to throw an error for this case

Reply:
If you run tar with a path like `./*` (eg `tar -czf my.tgz ./*`) then the filename inside the tar get prepended with `./`.